### PR TITLE
Update enable-the-report-message-add-in.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the document about Add-in being greyed out in case of a shared/group or delegated mailbox

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8020

File: microsoft-365/security/office-365-security/enable-the-report-message-add-in.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ If you're a global administrator or an Exchange Online administrator, and Exchan
   - Outlook included with Microsoft 365 apps for Enterprise
   - Outlook app for iOS and Android
 
-- Both add-ins are not available for shared mailboxes.
+- Both add-ins are not available for shared, group or delegated mailboxes (The add-in will be greyed out).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/enable-the-report-message-add-in.md]:
```suggestion
- Both add-ins are not available for shared, group, or delegated mailboxes (the add-ins will be greyed out).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@RatulaC Please review and merge this PR. Thanks

**Comment by yogkumgit** [comment]
@RatulaC Gentle reminder.

**Pull request merged**